### PR TITLE
www-client/chromium: DEPEND on freetype[png]

### PR DESCRIPTION
**Pull request by vilhelmgray**
Closes: https://bugs.gentoo.org/759256
Signed-off-by: William Breathitt Gray <vilhelm.gray@gmail.com>

### CODE REVIEW

File: www-client/chromium/chromium-87.0.4280.88.ebuild [hunk 1]
```diff
@@ -55,7 +55,7 @@ COMMON_DEPEND="
 	>=dev-libs/nss-3.26:=
 	>=media-libs/alsa-lib-1.0.19:=
 	media-libs/fontconfig:=
-	media-libs/freetype:=
+	media-libs/freetype[png]:=
```
Comment (telans) [inline www-client/chromium/chromium-87.0.4280.88.ebuild:58]:
```suggestion
	media-libs/freetype:=[png]
```

File: www-client/chromium/chromium-88.0.4324.27.ebuild [hunk 2]
```diff
@@ -55,7 +55,7 @@ COMMON_DEPEND="
 	>=dev-libs/nss-3.26:=
 	>=media-libs/alsa-lib-1.0.19:=
 	media-libs/fontconfig:=
-	media-libs/freetype:=
+	media-libs/freetype[png]:=
```
Comment (telans) [inline www-client/chromium/chromium-88.0.4324.27.ebuild:58]:
```suggestion
	media-libs/freetype:=[png]
```

File: www-client/chromium/chromium-89.0.4343.0.ebuild [hunk 3]
```diff
@@ -55,7 +55,7 @@ COMMON_DEPEND="
 	>=dev-libs/nss-3.26:=
 	>=media-libs/alsa-lib-1.0.19:=
 	media-libs/fontconfig:=
-	media-libs/freetype:=
+	media-libs/freetype[png]:=
```
Comment (telans) [inline www-client/chromium/chromium-89.0.4343.0.ebuild:58]:
```suggestion
	media-libs/freetype:=[png]
```

**Comment by vilhelmgray** [comment]
Closed in favor of https://github.com/gentoo/gentoo/pull/18593

**Pull request closed without merging**